config/config.go

```diff
@@ -1,2 +1,41 @@
 // Package config contains application configuration.
 package config
+
+import (
+	"strings"
+
+	"github.com/spf13/viper"
+)
+
+const (
+	configPath = "./"
+	configName = "viper.config"
+)
+
+type PostgresConfig struct {
```

Comment:
Config better to be moved to storage package and I suggest name DatabaseConfig.

Reply:
why config cant be in config folder?